make-rules/shared-macros.mk

```diff
@@ -100,6 +100,9 @@ space := $(empty) $(empty)
 
 ROOT =			/
 
+# Distribution version
+DISTRIBUTION_NAME = OpenIndiana
```

Comment:
So, question is if we should include 'Hipster' in DISTRIBUTION_NAME :)

Reply:
Yes I would say so. It makes us stand out from the older Release based OpenIndiana versions. Or we could follow OmniOS here.